src/odemis/acq/move.py

```diff
@@ -576,6 +586,7 @@ def _doCryoSwitchSamplePosition(future, target):
 
     try:
         if role == "enzel":
+            # TODO enzel code broken as rm is used in other functions insteas of rm
```

Comment:
is this comment line as it should? referring 2 times to 'rm'? There is also a typo here: `insteas of rm` should be `instead of rm`